plugins/openstack/keystone/check_keystone-api.sh

```diff
@@ -38,9 +38,10 @@ usage ()
     echo " -H <Auth URL>    URL for obtaining an auth token"
     echo " -U <username>    Username to use to get an auth token"
     echo " -P <password>    Password to use ro get an auth token"
+    echo " -T <seconds>     Timeout after integer <seconds>"
```

Comment:
It's not really a timeout. It's just a warning to indicate it took longer than expected.